ts/store/reducers/entities/services/readStateByServiceId.ts

```diff
@@ -24,6 +25,13 @@ export function readServicesByIdReducer(
   action: Action
 ): ReadStateByServicesId {
   switch (action.type) {
+    case getType(removeServiceTuples): {
+      const serviceTuples = action.payload;
+      const newState = { ...state };
+      // tslint:disable-next-line: no-object-mutation
+      serviceTuples.forEach(_ => delete newState[_.e1]);
```

Comment:
to avoid forEach/delete usage, consider using the reduce method also here

Reply:
before making the change I would also evaluate the performance